pyqtorch/core/measurement.py

```diff
@@ -12,28 +12,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from cgi import parse_multipart
+from typing import Any
+
 import torch
+
 from pyqtorch.core.operation import X, Y, Z
 
-qubit_operators = {'X': X, 'Y': Y, 'Z': Z}
+"""
+TODO: we should create the operator tensor only once, on startup in the network
+"""
```

Comment:
@awennersteen What do you mean exactly here? Do you want to initialize the predefined qubit operators at startup?

Reply:
yeah,
but I later realized that it is probably best kept out of the PyQ library

Reply:
@awennersteen Agreed.